pom.xml

```diff
@@ -196,6 +194,13 @@
         <groupId>org.jvnet.jaxb2.maven2</groupId>
         <artifactId>maven-jaxb2-plugin</artifactId>
         <version>${maven.jaxb2.plugin.ver}</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>2.3.5</version>
```

Comment:
@DmytroBykov1,  please move dependency version to properties.

Reply:
done